boot/freeldr/freeldr/arch/i386/machpc.c

```diff
@@ -636,34 +636,42 @@ DetectSerialPointerPeripheral(PCONFIGURATION_COMPONENT_DATA ControllerKey,
     }
 }
 
-static
+ULONG
+PcGetSerialPort(ULONG Index, PULONG Irq)
+{
+    ULONG PcIrq[MAX_COM_PORTS] = {4, 3, 4, 3};
```

Comment:
static variable.

Reply:
`static` is usually used for variables in case we want to initialize some of them in a function just once. Did you mean `const`? Also it was done like that previously: https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/i386/machpc.c;hb=0a8459767965cb948eb29303ef904002f3049e0a#l646

Reply:
https://stackoverflow.com/questions/13185751/what-is-the-difference-between-static-const-and-const/13185807

Reply:
Alright, I fixed it :sweat_smile: